z3c/dependencychecker/package.py

```diff
@@ -214,10 +215,11 @@ def set_declared_extras_dependencies(self):
     def analyze_package(self):
         top_folder = self.metadata.top_level
         for module_obj in MODULES:
-            logger.debug(module_obj)
+            logger.debug("Starting analyzing files using %s...", module_obj)
```

Comment:
On the next debug line you are already pointing which ``module_obj`` is being used, so the output now looks like:

```
DEBUG: Package name is plone.api
DEBUG: Starting analyzing files using <class 'z3c.dependencychecker.modules.PythonModule'>...
DEBUG: Searching dependencies (with PythonModule) in file /home/gil/Documents/plone/plone5/plone/src/plone.api/src/plone/__init__.py...
DEBUG: Searching dependencies (with PythonModule) in file /home/gil/Documents/plone/plone5/plone/src/plone.api/src/plone/api/env.py...
DEBUG:     Import found: AccessControl.SecurityManagement.getSecurityManager
```

IMHO this line is superfluous, as the information is already provided on the following lines. What do you think?

OTOH we could add some padding on the ``Search dependencies`` and double the padding on ``Import found`` so it could look like this:

```
DEBUG: Starting analyzing files using 'PythonModule'...
DEBUG:     Searching dependencies in file /home/gil/Documents/plone/plone5/plone/src/plone.api/src/plone/api/env.py...
DEBUG:         Import found: AccessControl.SecurityManagement.getSecurityManager
DEBUG:         Import found: AccessControl.SecurityManagement.getSecurityManager
DEBUG:         Import found: AccessControl.SecurityManagement.getSecurityManager
DEBUG:     Searching dependencies in file /home/gil/Documents/plone/plone5/plone/src/plone.api/src/plone/api/env.py...
DEBUG:         Import found: AccessControl.SecurityManagement.getSecurityManager
DEBUG: Starting analyzing files using 'ANOTHERMODULE'...
DEBUG:     Searching dependencies in file /home/gil/Documents/plone/plone5/plone/src/plone.api/src/plone/api/env.py...
```

In any case, we should change the ``module_obj`` here with ``module_obj.__name__`` as you did below.

Reply:
I don't want to get too clever with indentation. One level in one specific place is bad enough.

The line has use as it shows the flow of the program. I'd keep the long "class" string in that case. The reason: I could imagine us adding a plugin mechanism later on in this exact spot that lets people add custom readers.